app/Http/Controllers/CategoriesController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ValidateCategory;
+use App\Models\Categories;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class CategoriesController extends Controller
+{
+    public function index()
+    {
+        $categories = Categories::all();
+
+        return view('sections.categories.categories-page', ['categories' => $categories]);
+    }
+
+    public function createCategory()
+    {
+        return view('sections.categories.create-category');
+    }
+
+    public function store(ValidateCategory $request)
+    {
+        $request->validated();
+
+        $category = new Categories();
+        $category->name = $request->name;
+        $category->description = $request->description;
+        $category->slug = Str::slug($request->name);
+        $category->save();
+
+        return redirect()->route('categories.categories')->with('success', 'Categoria criada com successo');
+    }
+
+    public function show($id)
```

Comment:
É bom tu tipar os parâmetros dos métodos sempre que possível, pra tu e tua IDE saber que tipo de dado tal variável guarda.
Vale para os outros metodos
```suggestion
    public function show(int $id)
```